benchmarks/buiter_et_al_2016_jsg/exp_1.prm

```diff
@@ -0,0 +1,183 @@
+# This is the input file for the setup for the benchmark described in Buiter et al, 2016
+# Experiment 1 which simulates a stable wedge undergoing translation.
+# Experiment 2 can be found in $ASPECT_INSTALL/aspect/benchmarks/buiter_et_al_2008_jgr
+# Refer to the README file for details on experimental setup for this benchmark.
+
+
+#####  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 14400
+set Use years in output instead of seconds = false
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Nonlinear solver tolerance             = 1e-9
+set Max nonlinear iterations               = 25
```

Comment:
Does the nonlinear solver ever reach this value? We never really want this to happen, so my suggestion would be to remove this line altogether or increase the value to 50 or 100. The latter values are a really just a check to make sure the solver does not get stuck and if these limits are reached then the non-linear solver tolerance (or other model parameters) need to be adjusted.